test/index.js

```diff
@@ -1,2 +1,3 @@
+require('any-promise/register')('bluebird', {Promise: require('bluebird')});
```

Comment:
We could, but that would mean relying on `window.Promise` which is `undefined` for phantomjs. So it would just fail. Perhaps adding a comment about that?